protocol/testing/testnet/start.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -eo pipefail
+
+# This file is the startup script for the validator / full-nodes. Copies the correct binaries into
+# the validator home directories, and starts the node using `cosmovisor` to run `dydxprotocold`.
+# Any arguments passed into this script is forwarded to `cosmovisor`.
+# Example usage: ./start.sh run start --home chain/.dydx-1
+
+source "./vars.sh"
+
+# Set up CosmosVisor.
+for i in "${!MONIKERS[@]}"; do
+	VAL_HOME_DIR="$HOME/chain/.${MONIKERS[$i]}"
+	# Copy binaries for `cosmovisor` from the docker image into the home directory.
+	# Work-around to ensure docker volume contains the same binaries as the git repo.
+	cp -r "$HOME/cosmovisor" "$VAL_HOME_DIR/"
+done
+
+# Set up CosmosVisor for full-nodes.
+for i in $(seq 0 $LAST_FULL_NODE_INDEX); do
```

Comment:
Why is this 0 through 2? I thought there are only 2 full nodes?

Reply:
No harm in initializing more home directories